src/bundle/Resources/views/LinkManager/view.html.twig

```diff
@@ -33,7 +30,14 @@
             <tbody>
                 <tr>
                     <th>{{ 'url.label.address'|trans }}</th>
-                    <td><a href="{{ url.url }}" target="_blank">{{ url.url }}</a></td>
+                    <td>
+                        <a href="{{ url.url }}" target="_blank">{{ url.url }}</a>
```

Comment:
Self-nitpick: URL here should be also truncated.

Reply:
I'm sorry, but I can't agree. There should be at least one place where the user can see the whole link. Currently, we truncating URL on the list and in the header, so if I do it in the way you proposed, then a user will always have to enter edit mode to see the whole link :)

Reply:
This is important. It's actually one of the points I commented about regarding this spec.

Yes, the entire link should be visible _somewhere_. A tooltip would be sufficient, in my opinion, as a large URL will always overflow.

But in addition, I'd like to suggest (it can be an improvement) that the URL is truncated _in the middle_: https://example.com/...turkey-sandwich instead of https://example.com/recipes...

Reply:
@bdunogier yes, you are right, it will be better to truncate the URL in the middle. 
What do you think about this: https://github.com/dzango/TwigTruncateExtension?
Should we use it (and add another dependency) or write own Twig extension?